capitalize-and-filter.js

```diff
@@ -0,0 +1,8 @@
+const capitalizeAndFilter = (stringArr) => {
+
+  const filterArr = stringArr.filter(item => item[0].toLowerCase() !== 'f');
```

Comment:
Good job, but also wanted to see you try this with a for loop as well.   You can make a new function if you don't want to delete this one, to get that remaining point.